pkgs/development/libraries/gdal/default.nix

```diff
@@ -9,27 +9,17 @@ with stdenv.lib;
 
 stdenv.mkDerivation rec {
   pname = "gdal";
-  version = "3.0.1";
+  version = "3.0.3";
 
   src = fetchFromGitHub {
     owner = "OSGeo";
     repo = "gdal";
     rev = "v${version}";
-    sha256 = "04rraqhygv8b8fy87qvdhkgx87whby9n98p3gxqr7kdrfymwnh8l";
+    sha256 = "1rbyxmgmp27a5wvm4g70jr79bazhdl8q9rcch2b78m73njdv73xa";
   };
 
   sourceRoot = "source/gdal";
 
-  patches = [
-    ./001.3_0_1.darwin.patch
```

Comment:
Please also remove this file.

Reply:
Oops, missed that. :see_no_evil:

Done in e8aa909. Also mentioned this in the issue description.